### PR TITLE
NEW: Development documentation

### PR DESCRIPTION
**Pull request by sidmohanty11**
# Development documentation

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,2 +1,10 @@
 # EmbeddedChat
 An easy to use full-stack component  (ReactJS + backend behaviors)  embedding Rocket.Chat into your webapp
+
+## Development
```
Comment (Sing-Li) [inline README.md:4]: Please describe what this is.    Show a screen capture of how it looks like.

How will  the potential contributor be able to test his code?

How can this component be embedded into a web page or react app as it exists right now?

**Review by Sing-Li — CHANGES_REQUESTED** [review]

**Comment by Sing-Li** [comment]
Please submit the README.md update ALONE as a separate PR  (without any code).   I'll be able to test and review the instructions.    Hopefully helping out Rohan a bit on this.

**Comment by sidmohanty11** [comment]
@Sing-Li sure, I will update the whole README in this PR and when I'm done I'll tag you!

**Comment by sidmohanty11** [comment]
@Sing-Li as of now, can you please give a quick review of the documentation. I've added all the necessary steps that should be taken to set the component up.

The component hasn't been deployed yet, we can release it after the Auth and API get complete. After that `npm install ....` will work!

File: README.md [hunk 2]
```diff
@@ -1,2 +1,68 @@
 # EmbeddedChat
-An easy to use full-stack component  (ReactJS + backend behaviors)  embedding Rocket.Chat into your webapp
+
+An easy to use full-stack component (ReactJS + backend behaviors) embedding Rocket.Chat into your webapp.
+
+*EmbeddedChat is a full-stack React component node module of the RocketChat application that is fully configurable, extensible, and flexible for use. It is tightly bound with the RocketChat server using Rocket.Chat nodejs SDK and its UI using RocketChat's Fuselage Design System.*
+
+![embeddedchatwall](https://user-images.githubusercontent.com/73601258/178119162-ecabb9b7-e3ae-4c70-8ab2-f6c02856f4c6.png)
```
Comment (Sing-Li) [inline README.md:7]: Not a good screen capture for this purpose.

It looks no different than a non-embedded chat window.    Bringing up the bigger question of -- what is this project for?

Your screen capture should clearly show (ALL) the added values that this component bring to our community.    This picture *is* your THOUSAND WORDS.

**Review by Sing-Li — CHANGES_REQUESTED** [review]

File: README.md (hunk 2 above)
Comment (sidmohanty11) [reply README.md:7]: True. 100% agreed. Let me search a bit around for mockups that I can use. I will send some samples in the channel.

**Review by Sing-Li — APPROVED** [review]

**Pull request merged**